transform/event_origin_indexer.go

```diff
@@ -10,6 +10,12 @@ type EventOriginIndexer struct {
 	BlockIndexer blockIndexer
 }
 
+const (
+	DeliverTx  = "DeliverTx"
+	BeginBlock = "BeginBlock"
+	EndBlock   = "EndBlock"
+)
```

Comment:
This looks good but we could also define a type for this: https://www.sohamkamani.com/golang/enums/

Reply:
you mean like it is now @pienkowb ? (its been updated)

Reply:
I would also move this to `event_origin_filter.go`